pwnlib/protocols/adb/__init__.py

```diff
@@ -381,7 +382,7 @@ def list(self, path):
 
         Examples:
 
-            >>> pprint(AdbClient().list('/data/user'))
+            >>> pprint.pprint(AdbClient().list('/data/user'))
```

Comment:
Please leave only `pprint()`, see comments for `pwnlib/util/misc.py`

Reply:
Changed as requested.